modules/virt-configuring-vm-dpdk.adoc

```diff
@@ -0,0 +1,109 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-attaching-vm-to-sriov-network.adoc
+
+:_content-type: PROCEDURE
+[id="virt-configuring-vm-dpdk_{context}"]
+= Configuring a virtual machine for DPDK workloads
+
+You can run Data Packet Development Kit (DPDK) workloads on virtual machines (VMs) to achieve lower latency and higher throughput for faster packet processing in the user space. DPDK uses the SR-IOV network for hardware-based I/O sharing.
+
+.Prerequisites
+* Your cluster is configured to run DPDK workloads.
+* You have created a project on the cluster in which the VM is running.
+
+.Procedure
+. Edit the `VirtualMachine` manifest to include information about the SR-IOV network interface, CPU topology, CRI-O annotations, and huge pages:
++
+.Example `VirtualMachine` manifest
+[source,yaml]
+----
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  name: rhel-dpdk-vm
+spec:
+  running: true
+  template:
+    metadata:
+      annotations:
+        cpu-load-balancing.crio.io: disable <1>
+        cpu-quota.crio.io: disable <2>
+        irq-load-balancing.crio.io: disable <3>
+    spec:
+      domain:
+        cpu:
+          sockets: 1 <4>
+          cores: 5 <5>
+          threads: 2
+          dedicatedCpuPlacement: true
```

Comment:
There is a BZ we discovered after publishing the upstream docs. We can resolve https://bugzilla.redhat.com/show_bug.cgi?id=2184914 it here (or we can handle it in a follow-up PR).

```suggestion
          dedicatedCpuPlacement: true
          isolateEmulatorThread: true
```